### ISSUE TITLE
Radicale Web Interface icons and Apache proxy

### ISSUE THREAD

**Issue by fcorneli** [Kozea/Radicale#1497]
When using Radicale version 3.2.0 via an Apache proxy, it cannot download the icons within the Radicale Web Interface.
Apache proxy config:
```
RewriteEngine On
RewriteRule ^/calendar$ /calendar/ [R,L]

<Location "/calendar/">
    ProxyPass        http://localhost:5232/ retry=0
    ProxyPassReverse http://localhost:5232/
    RequestHeader    set X-Script-Name /calendar
</Location>
```
Within the web browser, it tries to download:
https://my_test_domain_here/calendar/.web/css/logo.svg
but cannot find this file. Same for all other icons.
However, functionally every button (login and such) works. Luckily there are tooltips so you can find out which missing icon performs which action.
Could this get fixed?

**Comment by fcorneli** [comment]
mmm... probably related to https://github.com/Kozea/Radicale/pull/1488

**Comment by pbiering** [comment]
should be fixed in master and upcoming 3.2.1
